model-lab/app/main.py

```
@@ -23,16 +29,31 @@
     allow_headers=["Content-Type"],
 )
 
+# ✅ Pre-load both model options at startup to avoid race conditions
+emotion_models = {
+    1: Model(model_path=MODEL_PATH, model_option=1),  # HuggingFace ViT
+    2: Model(model_path=MODEL_PATH, model_option=2)   # ONNX with CV2
+}
+
 @app.post("/predict")
 async def predict(file: UploadFile = File(...), model_option: int = Form(1)):
```

Ruff (0.14.5)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)